assets/minecraft/texts/splashes.txt

```diff
@@ -6,9 +6,95 @@ Made by Sake!
 Made by &flower!
 Made by Nexx!
 Made by Skye!
-Made by Laxla!
+Made by Laxystem!
+(We're not a laxlative :kittygun:)
 Made by Felix!
-Made by UnlikePaladin!
+Made by UnlikePaladin!ou
 Made by Ted!
 Made by Echo!
 Check out the Ko-Fi!
+pridecraft.gay
+No cishets were harmed in the making of this pack
+PRIDE
+You are valid.
+You are valid!
+YOU ARE VALID!!!
+people.forEach { it.isValid = true }
+--. .- -.--
+LESBIAN
+GAY
+BI
+TRANS
+QUEER
+ACE
+(The I requested to leave. Respect that!)
+LGBTQA+!
+WE'RE WHO WE'RE!
+Proud Edition
+🏳️‍⚧️
+🏳️‍🌈
+Different is beautifaul!
+FLAGXPLOSION!
+TODO: be more gay
+smh this ain't gay enough
+the §fg§r in §fgay§r stands for §fgood§r
+the §fa§r in §fgay§r stands for §fhey§r
+the §fy§r in §fgay§r stands for something, idk really
+:3
+:33333333
+:3333333333333333333
+u r wewy cute 2day
+LOVING that outfit of yours
+Damn, that's a good skin
+We're QUEER and WE'RE PROUD of it.
+100% Queer
+Queerium Tanks: 100% Full
+That's a damn good resource pack over there
+we love what you did to the place
+You're an AMAZING §fredstoner§r!
+You're an AMAZING §fbuilder§r!
+You're an AMAZING §fexplorer§r!
+You're an AMAZING §ffighter§r!
+You're an AMAZING §fresource gatherer§r!
+You're an AMAZING §fmodder§r!
+You're an AMAZING §ffarmer§r!
+You're an AMAZING §ftrader§r!
+You're an AMAZING §fperson§r!
+You're AMAZING and you're VALID!
+You're AMAZING and you're VALID and whoever thinks otherwise is just WRONG!
+Join the discord.pridecraft.gay!
+Includes Apache Helicopters! (not an official PrideCraft™ business statement)
+APACHE HELICOPTERS ARE VALID TOO!
+sorry for the capslock, we blame the kybord. It is now vanned.
+No Queer's Sky /ref
+Works w/ Fabric!
+Works w/ Forge!
+Works w/ Quilt!
+Works w/ Neoforge!
+Works w/ Vanilla!
+Three gay people entered a bar, and t'was really proud.
+Check out PrideMod!
+Check out PridePack x32!
+Check out PrideCraft!
+Quilt and Queer both have a Q... conspiracy?
+THE ENTIRE GAME IS GAY MUHAHAHAHAHAHAHA
+The §fg§r in §fgame§r stands for §fgay§r
+The §fg§r in §fgay§r stands for §fgame§r
+Not enough §fsnow§r.
+Check out §flolcat§r on Linux. Guaranteed Queerfullness.
+This town ain't small enough for the two of us. I mean, we gotta meet each other more, it's just huge, innit?
+This is so gay, we're in heaven.
+RGB lighting is actually a queer conspiracy to secretely brainwash all gamers in the world into gayness.
+C? Trans!
+Truly a masterpiece. Of the cake. Of the coming out party! Yey!
+Bravo, Captain Holt! 🤖
+Mrow
+Meow
+Cats are da best! Lolcat! Proof: there's a linux command named lolcat that gays cat.
+Being gay is a way of life.
+Also, it's just really fun :3
```

Comment:
Does this refer to Line 94 or the game? Only one line will display at once

Reply:
both